docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+
+The following sections will cover *Style by value* with different property types. For example, when dealing with a transaction dataset we could style by numeric data like the amount of each payment, or by categorical data like the method of payment (credit card, cash...).
+
+#### Numerical properties
+
+##### Showing unclassified numerical data
+
+Going back to our previous example, it's common to want to map a continuous range of numeric data like population density data, to a continuous range of colors, for example, the range of colors between black and yellow.
+
+This is very easy to do with CARTO VL, as shown before you just need to use:
+ ```CARTOVL_Viz
+color: ramp($population_density, [black, yellow])
+ ```
+
+ This will map the feature with the lowest population density in the Source data to *black* and the feature with the highest population density to *yellow*. You can even set intermediate colors in the color list like `[black, gray, yellow]`.
+
+ Matching the input with the context of the lowest population density and highest population density is actually done by the [`linear`](https://carto.com/developers/carto-vl/reference/#cartoexpressionslinear) function, which is placed automatically by `ramp` when the input is a numeric property. CARTO VL `ramp` function just transforms `ramp($population_density, [green, yellow, red])` to `ramp(linear($population_density), [green, yellow, red])`. These transformations are what we call *implicit casts* and are a common topic in CARTO VL.
+
+##### Overriding the default range and avoiding outliers
+
+ Let's see another *implicit cast*, this time one a little bit more interesting.
+
+ The [`linear`](https://carto.com/developers/carto-vl/reference/#cartoexpressionslinear) function has another *implicit cast*. When linear is called with only one parameter it will transform things like `linear($population_density)` to things like `linear($population_density, globalMin($population_density), globalMax($population_density))`. The second and third parameter of `linear` is what sets the context of the lowest and highest population densities for `ramp`.
+
+ Sometimes, the data has [outliers](https://en.wikipedia.org/wiki/Outlier): features with data that is very far away from the norm. In these cases, we may want to ignore them when computing the ramp. This can be easily done by manually setting the second and third parameters of linear to the minimum and maximum values of the data range we are interested.
+
+Let's see an example with *implicit casts* and explicit linear range.
+
+<div class="example-map" style="margin: 20px auto !important">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/ramp/population-density-basic.html"
```

Comment:
Nice example, very illustrative. 

I would try to adjust a bit the labels, as the expressions do not align with the input's circle. Maybe it's just easier to give them a shorter title (like Expression 1, Expression 2...) and include the expression itself below, with its explanation

Reply:
I hope it looks better with the real CSS on production